Useful dev tools section (#287)
ethereum-optimism/community-hub: src/docs/build/monitoring.md

---
title: Monitoring
lang: en-US
---

# {{ $frontmatter.title }}

## Status Page

> [Optimism](https://optimism.io) has an [instatus Status page](https://optimism.instatus.com/). This page includes public APIs, the gateway, deposits, withdraws and transaction sequencing for both Optimism mainnet and Optimistic Kovan.

Reviewer: ```suggestion
Check out the [Optimism status page](https://optimism.instatus.com/).
This page includes public APIs, the gateway, deposits, withdraws and transaction sequencing for both Optimism mainnet and Optimistic Kovan.
```
